packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -253,6 +253,7 @@ export const prepareDiagnosticMetricItem = (webex: any, item: any) => {
       joinTimes.meetingInfoReqResp = cdl.getMeetingInfoReqResp();
       joinTimes.showInterstitialTime = cdl.getShowInterstitialTime();
       joinTimes.registerWDMDeviceJMT = cdl.getRegisterWDMDeviceJMT();
+      joinTimes.getU2CTime = cdl.getU2CTime();
```

Comment:
suggestion: There are no tests that actually check this method (`getU2CTime`) is called. Would be nice to have that, despite nothing else here being tested properly.